vmware-event-router/internal/processor/aws_event_bridge.go

```diff
@@ -115,7 +115,7 @@ func NewAWSEventBridgeProcessor(ctx context.Context, cfg connection.Config, sour
 			return nil, errors.Wrap(err, "could not list event bridge rules")
 		}
 
-		arnLoop:
+	arnLoop:
```

Comment:
Assuming this was a formatting issue on my end, correct?

Reply:
I have formatter on save, probably this is the right formatting though I am not sure.